lib/jekyll/post.rb

```diff
@@ -118,6 +119,14 @@ def title
       data.fetch('title') { titleized_slug }
     end
 
+    # Public: the Post excerpt_separator, from the YAML Front-Matter or site default
+    #         excerpt_separator value
+    #
+    # Returns the post excerpt_separator
+    def excerpt_separator
+      data.fetch('excerpt_separator') { site.config['excerpt_separator'].to_s }
```

Comment:
@parkr yes, but I believe in that context they will have the similar behaviour, because I think, no one will define `excerpt_separator` as a `nil` or `false`. And in my case the all the jekyll test results dont change the state after applying this change =)